tendrl/node_agent/provisioner/ceph/utils.py

```diff
@@ -0,0 +1,16 @@
+import os
+import struct
+import time
+import base64
+
+
+def generate_auth_key():
```

Comment:
please add comment explaining what type of key is required to be generated and how it is generated